README.md

```diff
@@ -1,27 +1,51 @@
-Sinatra-based Identity SP
-=========================
+identity-saml-sinatra
+=====================
 
-[![CircleCI](https://circleci.com/gh/18F/identity-sp-sinatra.svg?style=svg)](https://circleci.com/gh/18F/identity-sp-sinatra)
+[![CircleCI](https://circleci.com/gh/18F/identity-saml-sinatra.svg?style=svg)](https://circleci.com/gh/18F/identity-sp-sinatra)
 
-Example service provide (SP) app for use with 18F's IdP.
+An example service provider application for use with [Login.gov](https://login.gov/)'s identity provider.
 
-These instructions assume [`identity-idp`](https://github.com/18F/identity-idp) is also running locally at `http://localhost:3000`. This sample sp is configured to run on `http://localhost:4567`.
+## Running locally
 
-### Setup
+These instructions assume [`identity-idp`](https://github.com/18F/identity-idp) is also running locally at http://localhost:3000 .
 
-    $ make setup
+1. Set up the environment with:
 
-### Testing
+  ```
+  $ make setup
+  ```
 
-    $ make test
+2. Generate a new key and self-signed certificate
 
-### Running (local development mode)
+  ```
+  $ openssl req -days 3650 -newkey rsa:2048 -nodes -keyout config/demo_sp.key \
```

Comment:
should we move this to a makefile command?

Reply:
> should we move this to a makefile command?

Yeah, if I follow correctly, this is a requirement for running the server, so we should be able to include it as part of `make setup`. I'll see to it.

Reply:
Oh, I think I misunderstood. That certificate is version controlled. I'm not really clear if and when it's supposed to be updated. It might still be better to make this local-only, and per above generate as part of `make setup`?

Reply:
Meh, let's leave it out for now and if it expires in 10 years and we need to remake it, the instructions will be in git history

Reply:
> Meh, let's leave it out for now and if it expires in 10 years and we need to remake it, the instructions will be in git history

Works for me 👍  Removed in e98a0cc.